spec/runtime/inline_spec.rb

```diff
@@ -193,4 +193,38 @@ def confirm(msg, newline = nil)
     expect(err).to be_empty
     expect(exitstatus).to be_zero if exitstatus
   end
+
+  it "installs inline gems when a Gemfile.lock is present" do
+    gemfile <<-G
+      source "https://rubygems.org"
+      gem "rake"
+    G
+
+    lockfile <<-G
+      GEM
+        remote: https://rubygems.org/
+        specs:
+          rake (11.3.0)
+
+      PLATFORMS
+        ruby
+
+      DEPENDENCIES
+        rake
+
+      BUNDLED WITH
+         1.13.6
+    G
+
+    script <<-RUBY
+      gemfile do
+        source "https://rubygems.org"
```

Comment:
this can't pull from an internet source, it should use one of the sources we build in the specs